src/util/asset/assetUtils.ts

```diff
@@ -73,3 +73,10 @@ export function sortAssetIds(asset1ID: number, asset2ID: number): number[] {
 export function isAlgo(id: number | string) {
   return Number(id) === ALGO_ASSET_ID;
 }
+
+/**
+ * Returns asset.asset_id
+ */
```

Comment:
```suggestion
/**
 * @returns Asset id as a number
 */
```
I believe comments shouldn't depend on the code implementation, they should explain what the function does

Reply:
I copied it directly from client code :D